extensions/wikia/CommunityPage/models/CommunityPageSpecialHelpModel.class.php

```diff
@@ -4,14 +4,30 @@ class CommunityPageSpecialHelpModel {
 	public function getData() {
 		return [
 			'title' => wfMessage( 'communitypage-help-module-title' )->plain(),
-			'link' => $this->getHelpPageLink(),
-			'linkText' => wfMessage( 'communitypage-help-module-link-text' )->plain()
+			'editPage' => wfMessage( 'communitypage-help-edit-page' )->plain(),
+			'addLinks' => wfMessage( 'communitypage-help-add-link' )->plain(),
+			'addNewPage' => wfMessage( 'communitypage-help-add-new-page' )->plain(),
+			'editPageLink' => $this->geteditPageLink(),
+			'addLinksPageLink' => $this->getAddLinksPageLink(),
+			'addNewPageLink' => $this->getAddNewPageLink()
 		];
 	}
+	
+	private function geteditPageLink() {
```

Comment:
What do you think about creating one method and pass message key, for example `getHelpPageLink(messageKey`)?